src/components/calcite-combobox/calcite-combobox.scss

```diff
@@ -137,21 +138,19 @@
 }
 
 .list-container {
-  overflow-y: auto;
-  max-height: 100vh;
+  @apply overflow-y-auto
+    max-h-screen
+    bg-foreground-1;
   width: var(--calcite-dropdown-width);
   background: var(--calcite-ui-foreground-1);
```

Comment:
you have to delete this line here...

Reply:
darn it, forgot to delete it after tailwindyfing it! DONE!